.npmrc

```diff
@@ -1,6 +1,8 @@
-@asl:registry = https://artifactory.digital.homeoffice.gov.uk/artifactory/api/npm/npm-virtual/
+registry=https://registry.npmjs.org/
```

Comment:
mega nit but this isn't necessary (npm is configured to use the npm public registry at https://registry.npmjs.org/ by default.)